app.js

```diff
@@ -63,4 +63,274 @@ function fillData() {
   textArea.value = desirializedObject.textArea;
 }
 
-window.addEventListener('load', fillData);
\ No newline at end of file
+window.addEventListener('load', fillData);
+
+// create the card information
+const otherProjects = [
+  {
+    name: 'Multi-Post Stories',
+    description:
+      "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.",
+    image: './assets/first-card.png',
+    desktopImg: './assets/large.png',
+    technologies: ['css', 'html', 'bootstrap', 'Ruby'],
+    liveLink: '#',
+    sourceLink: '#',
+  },
```

Comment:
- [ ] Good job in implementing the suggestions from the previous reviewer. However, there is still one issue you need to fix on this part of the project. Some content of the page cut off at the bottom of your modal on the mobile. Kindly look into it and fix it.

### Your Design
![mobile-modal](https://user-images.githubusercontent.com/26597432/202853197-b6003028-2245-4ce4-af30-95986737d1ba.png)